setup.py

```diff
@@ -40,13 +40,15 @@
 kerberos_requires = ['requests-kerberos']
 visualization_requires = ['graphviz']
 pandas_requires = ['multipledispatch']
+turbodbc_requires = ['pybind11', 'pyarrow', 'turbodbc>=2.0.0']
```

Comment:
Is `pybind11` necessary?

Reply:
It is when I tested it back then.

Reply:
As long as `turbodbc` does not ship wheels on all platforms, you might need it with some `setuptools` versions (older ones). Note that there is the extra-requirement `[arrow]` so that `turbodbc[arrow]` will already pull in `pyarrow`.

Reply:
okay got it.